app/build.gradle

```diff
@@ -93,7 +93,7 @@ android {
             buildConfigField "boolean", "PAID_OOTT", "false"
             dimension "tier"
 
-            minSdkVersion 21
+            minSdkVersion 23
```

Comment:
I would like to keep the minSdkVersion on 21 (Android 5.0) at the moment.. 
Is this really needed to update to 23??

Reply:
Yes unfortunately the documentations say:
Android Auto is only compatible with phones running Android 6.0 (API level 23) or higher.

Reply:
I think that it just doesn't recognize the service on lower android sdks.. so its just not there on lower version.
I think that it doesnt crash, so Android 5 should still be possible for the whole app

Reply:
I had an error at build time with sdk version 21, but maybe there is a workaround.. .

Reply:
The error is:
```
uses-sdk:minSdkVersion 21 cannot be smaller than version 23 declared in library [com.google.android.libraries.car:car-app:1.0.0-beta.1] /home/florian/.gradle/caches/transforms-2/files-2.1/7e90d133bb43a13c15968d3cd3f02db3/jetified-car-app-1.0.0-beta.1/AndroidManifest.xml as the library might be using APIs not available in 21
Suggestion: use a compatible library with a minSdk of at most 21,
		or increase this project's minSdk version to at least 23,
		or use tools:overrideLibrary="com.google.android.libraries.car.app" to force usage (may lead to runtime failures)
```
Apparently it is possible to force the build with : `tools:overrideLibrary="com.google.android.libraries.car.app"`

Reply:
Maybe it would be OK to move to the min SDK Version 23:
https://www.xda-developers.com/android-version-distribution-statistics-android-studio/